### PR TITLE
refactor(telecom): kill v7 tucVoipService.fetchAll 

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-8893
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [ ] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js [hunk 1]
```diff
@@ -260,39 +253,58 @@ export default class TelecomTelephonyAliasConfigurationAgentsOvhPabxCtrl {
     }
   }
 
-  getInternalNumber(newNumber) {
-    const allNumbers = flatten(map(this.groupList, 'services'));
-    const [, numberSuffix] = newNumber.replace(/ /g, '').split(NUMBER_PREFIXES);
-    return allNumbers.find(({ serviceName }) =>
-      endsWith(serviceName, numberSuffix),
-    );
+  async getInternalNumber(newNumber) {
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js:256]: Are we sure async is not causing any trouble with angularJS $digest cycle ?

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/rule.controller.js [hunk 2]
```diff
@@ -69,10 +71,18 @@ export default class DialplanExtensionRuleCtrl {
...
-  getVoicemailInfos() {
-    return this.TelephonyMediator.findService(
-      this.getRuleAttribute('actionParam'),
-    );
+  fetchVoicemailInfos() {
+    if (this.isLoadingVoicemailInfos) {
+      return;
+    }
+    this.isLoadingVoicemailInfos = true;
+    this.TelephonyMediator.findService(this.getRuleAttribute('actionParam'))
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/rule.controller.js:79]:
```suggestion
    return this.TelephonyMediator.findService(this.getRuleAttribute('actionParam'))
```

**Review by cbourgois — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js [hunk 3]
```diff
@@ -261,11 +254,8 @@ export default class TelecomTelephonyAliasConfigurationAgentsOvhPabxCtrl {
   }
 
   getInternalNumber(newNumber) {
-    const allNumbers = flatten(map(this.groupList, 'services'));
-    const [, numberSuffix] = newNumber.replace(/ /g, '').split(NUMBER_PREFIXES);
-    return allNumbers.find(({ serviceName }) =>
-      endsWith(serviceName, numberSuffix),
-    );
+    const [, axiom] = newNumber.replace(/ /g, '').split(NUMBER_PREFIXES);
```
Comment (frenautvh) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js:257]: nitpick : `/\s/g`

**Review by frenautvh — APPROVED** [review]

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js (hunk 1 above)
Comment (y4nnL) [reply packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js]: It worked surprisingly well but I've removed it anyway

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**